### PR TITLE
Added the storage mount command implementation

### PR DESCRIPTION
**Pull request by mik-dass**
Signed-off-by: mik-dass mrinald7@gmail.com
Issue : #229

### CODE REVIEW

File: pkg/storage/storage.go [hunk 1]
```diff
@@ -264,3 +264,37 @@ func GetMountPath(client *occlient.Client, applicationName string, componentName
...
+		return errors.Wrap(err, "unable to get the component name associated with the storage")
+	}
+	if storageComponent != "" {
+		return fmt.Errorf("the given storage is already mounted to the component %v", storageComponent)
+	}
+	pvc, err := client.GetPVCFromName(generatePVCNameFromStorageName(storageName))
+	if err != nil {
+		return errors.Wrap(err, "unable to get the pvc from the storage name")
+	}
+	err = client.UpdatePVCLabels(pvc, storagelabels.GetLabels(storageName, componentName, applicationName, true))
```
Comment (kadel) [inline pkg/storage/storage.go]: PVC labels should be updated only after the volume was successfully added to the DC.
This should be the last thing that is done in this function.

**Comment by mik-dass** [comment]
@kadel Done

File: cmd/storage.go [hunk 2]
```diff
@@ -172,17 +172,57 @@ var storageListCmd = &cobra.Command{
 	},
 }
 
+var storageMountCmd = &cobra.Command{
+	Use:   "mount [storage name]",
+	Short: "mount storage to a component",
+	Example: `
+  # Mount storage 'dbstorage' to current component
```
Comment (cdrage) [inline cmd/storage.go]: Please move this line above so it's in-line with `Example:` or else it get's an extra space within help.

```
	Example: `  # Mount storage 'dbstorage' to current component`
```

Like that ^^

**Comment by kadel** [comment]
@mik-dass this needs rebase

**Comment by mik-dass** [comment]
@kadel Done

**Comment by kadel** [comment]
> one small change, otherwise, LGTM

 @cdrage, this was addressed. I'll merge this.

**Pull request merged**